index.src.html

```diff
@@ -407,26 +438,32 @@ <h3 dfn export id="referrer-policy-origin-when-cross-origin" oldids="referrer-po
     <a>potentially trustworthy URL</a>s.
   </div>
 
+  <div class="example">
+    If a document a <code>https://example-1.com</code> sets a policy of
+    <a>"<code>origin-when-cross-origin</code>"</a>, and fetches a module script
+    at <code>https://example-2.com</code>, which then fetches a descendant script at
+    <code>https://example-1.com/descendant.js</code>, the request to the descendant
+    script would send a <a><code>Referer</code></a> header with a value of
+    <code>https://example-1.com/</code>.
+  </div>
```

Comment:
It might be worth having another example where the document is example-1.com, the module script is example-2.com, and the descendant script is also example-2.com

Reply:
Done.